iwyu_output.h

```diff
@@ -208,6 +213,8 @@ class OneIncludeOrForwardDeclareLine {
   int end_linenum_;
   bool is_desired_;                 // IWYU will recommend this line
   bool is_present_;                 // line was present before the IWYU run
+  bool is_elaborated_type_ =  // Fwd-decl is introduced by elaborated type.
```

Comment:
I wonder if it would play out nicer if you did a commit first with the member initializers, and removed all the long initializer lists from the `OneIncludeOrForwardDeclareLine` constructors? That way it's fully consistent every step of the way.

Reply:
Actually, let's do initializers and format in the same commit, and in a separate PR. It's a nice step improvement.